src/template-tags/tickets.php

```diff
@@ -822,8 +824,14 @@ function tribe_tickets_get_event_ids( $id ) {
 	function tribe_tickets_get_ticket_provider( $id ) {
 		/** @var Tribe__Tickets__Data_API $data_api */
 		$data_api = tribe( 'tickets.data_api' );
+		$provider = $data_api->get_ticket_provider( $id );
+		
+		// Skip if the provider is Tickets Commerce but the module is disabled.
+		if ( $provider instanceof TEC\Tickets\Commerce\Module && ! tec_tickets_commerce_is_enabled() ) {
```

Comment:
Minor: You should add a `use` statement to the top of the file for `TEC\Tickets\Commerce\Module`, and then you can use the shortened name here instead of the fully-qualified name.